### ISSUE TITLE
Outreachy Documentation Project: <Sayantani Saha/sayantani11>

### ISSUE THREAD

**Issue by sayantani11** [ersilia-os/ersilia#101]
Applicant: <@sayantani11>
  
Welcome to the Ersilia Open Source Initiative. This issue will serve to track all your contributions for the project “Improve the documentation and outreach material of the Ersilia Model Hub”.
  
Please tick the tasks as you complete them. To make a final application it is *not* required to have completed all tasks. Only the *Initial Steps* and *Community* sections are REQUIRED. The tasks are not ordered from more to less important, they are simply related to different skills. Start where you feel most comfortable. This project can be adapted to the applicants interests, please focus on the type of tasks that you prefer / have better skills / would like to work on as an intern.

---
### Initial steps:
- [x] Record your application for the project in the outreachy website referencing this issue. Please make sure to select the right project on the website.
- [x] Join the Slack channel to follow public communications
- [x] Comment under this issue explaining why are you interested in this project
---
### GitHub documentation:
- [ ] Create a README file with the name <README_yourname.md> under the /documentation folder
- [ ] Link the #PR in a comment under this issue
- [ ] Incorporate feedback from the mentor
---
### Writing dissemination material
- [x] Read the [Strategic plan 2021-2023](https://ersilia.gitbook.io/ersilia-wiki/) for Ersilia and create a 1-page blogpost with the main points
- [x] Comment under this issue with a link to the blogpost (a google docs for example)
- [x] Incorporate feedback from the mentor
- [ ] Choose your own topic related to Ersilia (AI/ML for biomedical research, neglected diseases, drug discovery…) and write a 1-page blogpost to communicate to a non-expert audience
- [ ] Comment under this issue with a link to the blogpost (a google docs for example)
- [ ] Incorporate feedback from the mentor
- [ ] Create a template for a twitter post to release every time a new model is incorporated in the Hub (twitter: 280 characters, you can suggest a main post +  thread with extra information) and add it as a comment under this issue
- [ ] Create a template short Newsletter (1 paragraph) to send every month to our community (funders, users, contributors). It should mention metrics (models in the hub, number of users, funding…), thank you etc 
---
### Technical skills (required for the tutorial only)
- [ ] [Install](https://ersilia.gitbook.io/ersilia-book/quick-start/installation) the Ersilia Model Hub
- [ ] Test one model
- [ ] Add a screenshot under this issue showing the model running in your computer
- [ ] Write a docstring for the ErsiliaModel class. Use the Google Python Style guide. Paste the docstring as a comment below (do not use a PR).
---
### Graphic material
- [x] Read the [Ersilia Brand Guidelines](https://medium.com/ersiliaio/ersilia-open-source-initiative-brand-guidelines-130fe53104f5)
- [x] Read [“Why Ersilia?”](https://ersilia.gitbook.io/ersilia-wiki/)
- [x] Create one image / slide to explain Ersilia’s mission and vision
- [x] Link to the image/slide as a comment under this issue
- [x] Incorporate feedback from the mentor
- [ ] Create two slides / short video showing how to use the Ersilia Model Hub and add them under the /tutorial folder 
- [ ] Link the #PR in this issue
- [ ] Incorporate feedback from the mentor
---
### Scientific content
- [x] Check the models available in the [Hub](https://airtable.com/shrXfZ8pqro0jjcsG/tblZGe2a2XeBxrEHP/viwd5XJVLslkE11Tg)
- [x] Select one model from the list and write a technical card (what is the model for, what input, which data was used to create it, what kind of ML algorithm uses…) for it
- [x] Add your card as a comment to this issue
- [ ] Search the scientific literature and suggest 3 new models (comment in this issue) that would be relevant to incorporate in the Hub.
---
### Community
- [ ] Look up two other projects and comment on their issues with feedback on one of their tasks
- [ ] If you have feedback from your peers, answer it in this issue.
---
### Other
If you have interest in working on related topics, or have new suggestions, please do the following
- [ ] Add a comment in this issue with your new idea, tagging the mentor 
- [ ] Get feedback from the mentor and act accordingly
- [x] Link in the comments any other PR you have contributed to.
---
### Final application
- [x] I have answered all comments from mentors and contributors
- [x] All PR or issues assigned to me are complete
- [x] I have submitted my final application to the project

**Comment by sayantani11** [comment]
Ersilia is an open source model that deals with drug discovery. The most interesting part of the project is that it deals with neglected diseases and help in discovery of the drugs related to that and also adverse effects. A major part of the project deals with documentation, as I have had contributed documentation in lot of projects previously and have experience in writing whitepaper as well, this project interests me keeping all this in mind. I hope I'll be able to add value to the project.

**Comment by GemmaTuron** [comment]
Hi @sayantani11 

Since you have done already some contributions not listed here, can I ask you to check @ifeoluwafavour issue and see how she has linked the previous work and do the same? To have everything in one place!

Thanks

**Comment by sayantani11** [comment]
@GemmaTuron Yes sure!!!

**Comment by sayantani11** [comment]
Contributions:
- The project was missing a contribution guildelines as of which made an issue to track that:
   https://github.com/ersilia-os/ersilia/issues/21

- Starting with open source can always be over whelming as of which made an issue and PR to add a readme that will be helpful for the new contributors:
   https://github.com/ersilia-os/ersilia/pull/70
   https://github.com/ersilia-os/ersilia/issues/44

- Ersilia deals with documentation, it will be a great feature if we could add a writing style guide then we will be able to maintain consistency, made an issue on that:
  https://github.com/ersilia-os/ersilia/issues/88

**Comment by Abhijay007** [comment]
Contributions:

* The project was missing a contribution guildelines as of which made an issue to track that:
   [Create a contributing guidelines for the project #21](https://github.com/ersilia-os/ersilia/issues/21)
 * Starting with open source can always be over whelming as of which made an issue and PR to add a readme that will be helpful for the new contributors:
   [guidelines for new contributors #70](https://github.com/ersilia-os/ersilia/pull/70)
   [Guidelines for onboarding of new contributors #44](https://github.com/ersilia-os/ersilia/issues/44)
 * Ersilia deals with documentation, it will be a great feature if we could add a writing style guide then we will be able to maintain consistency, made an issue on that:
   [Create a writing style guidelines for ersilia #88](https://github.com/ersilia-os/ersilia/issues/88)

**Comment by sayantani11** [comment]
- Added a writing style guideline that will be there for contributors to be followed while making documentation changes to Ersilia.
   https://github.com/ersilia-os/ersilia/pull/152

- Ersilia deals with a lot of documentation, keeping that in mind maintaining consistency across all the files is very important. And it's manually impossible to check it for every PR. So adding workflows or test will be a great addition that will reduce work for the maintainers and improve efficiency
  https://github.com/ersilia-os/ersilia/issues/177
Added a PR for the above issue
https://github.com/ersilia-os/ersilia/pull/210

- To make things easier for everyone, it was decided that the new contributors file will be merged with the contributions guideline file, so made a PR for that.
 https://github.com/ersilia-os/ersilia/pull/199

**Comment by sayantani11** [comment]
Made an feature request that can help contributors raise WIP PRs
https://github.com/ersilia-os/ersilia/issues/246

**Comment by sayantani11** [comment]
[slide on EOSI's mission and vision](https://docs.google.com/presentation/d/11DGunZOnNU7Fd05bDaGcBx4OvIqKdydMB8HcO5fK5s0/edit?usp=sharing)

**Comment by sayantani11** [comment]
- Made an feature request that can help contributors raise WIP PRs
   https://github.com/ersilia-os/ersilia/issues/246
- 1 page blog on Strategic plan 2021-2023
   [Strategic plan summary](https://docs.google.com/document/d/18ws_4hmSmZT7NjCx2e7XCXGVhDadAQUrdRKXqEN3La8/edit?usp=sharing)
- Image/ slide on EOSI's mission and vision
  [Mission & Visio](- Made an feature request that can help contributors raise WIP PRs
   https://github.com/ersilia-os/ersilia/issues/246
- 1 page blog on Strategic plan 2021-2023
   [Strategic plan summary](https://docs.google.com/document/d/18ws_4hmSmZT7NjCx2e7XCXGVhDadAQUrdRKXqEN3La8/edit?usp=sharing)

**Comment by sayantani11** [comment]
- Made an feature request that can help contributors raise WIP PRs
   https://github.com/ersilia-os/ersilia/issues/246
- 1 page blog on Strategic plan 2021-2023
   [Strategic plan summary](https://docs.google.com/document/d/18ws_4hmSmZT7NjCx2e7XCXGVhDadAQUrdRKXqEN3La8/edit?usp=sharing)
- Slide/ image on EOSI's mission and vision
  [Mission & Vision](https://docs.google.com/presentation/d/11DGunZOnNU7Fd05bDaGcBx4OvIqKdydMB8HcO5fK5s0/edit?usp=sharing)

**Comment by sayantani11** [comment]
### **TECHNICAL CARD**

**Identification**
_Model ID:_ eos6tg8
_Name:_ Natural product fingerprint
_Description:_ 
A structural fingerprint based on natural products

**General info**
- Built using Pytorch
- the ADAM algorithm in combination with the One Cycle Learning Rate policy is used for Training
_Input:_ Compound
_Output:_ Vector
_Mode:_ Pretrained
_Status:_ Ready

**Resources**
_License:_ MIT
_Github:_ 
https://github.com/ersilia-os/eos6tg8
_Source:_ 
https://github.com/kochgroup/neural_npfp
_Publication:_ https://www.sciencedirect.com/science/article/pii/S2001037021003226?via%3Dihub#f0010

**Comment by sayantani11** [comment]
@GemmaTuron Can you review the work? And @miquelduranfrigola for #210 can you please rename the branch to main?

**Comment by GemmaTuron** [comment]
> ### **TECHNICAL CARD**
> **Identification** **_Model ID:_** eos6tg8 **_Name:_** Natural product fingerprint **_Description:_** A structural fingerprint based on natural products
> 
> **General info**
> 
> * Built using Pytorch
> * the ADAM algorithm in combination with the One Cycle Learning Rate policy is used for Training
> 
> **_Input:_** Compound **_Output:_** Vector **_Mode:_** Pretrained **_Status:_** Ready
> 
> **Resources** **_License:_** MIT **_Github:_** https://github.com/ersilia-os/eos6tg8 **_Source:_** https://github.com/kochgroup/neural_npfp **_Publication:_** https://www.sciencedirect.com/science/article/pii/S2001037021003226?via%3Dihub#f0010

Hi @sayantani11 

this looks good, I would only perhaps work a bit on the general info section, as most users won't be experts so that it is built with PyTorch or another one its not so relevant to them.
Otherwise great job!

**Comment by GemmaTuron** [comment]
The Outreachy contribution period has come to an end, so I will close this issue. If you want to continue contributing to Ersilia, please follow the guidelines in the CONTRIBUTING file
